src/interfaces/todo-item-interface.ts

```diff
@@ -48,5 +48,4 @@ export interface TodoItemIF {
 
   // alterations only
   legalType?: string
-  goodStanding?: boolean
```

Comment:
This was a business state flag, not a todo item flag. So I removed this and just use the store getter instead.